dist/common_functions.sh

```diff
@@ -42,7 +42,7 @@ while len(pids):
 }
 
 # Do normal cleanup on exit or abort.
-setup_trap() { trap 'kill `allchildpids` 2>/dev/null; rm -f $t' 0 1 2 3 13 15; }
+setup_trap() { trap 'kill `allchildpids` 2>/dev/null; rm -f $t*; '"$@" 0 1 2 3 13 15; }
```

Comment:
Can this be changed to make sure `$t` is not empty please? I don't want this to turn into `rm -f *` due to a bug or misuse or whatever.